src/track/trackmetadatataglib.cpp

```diff
@@ -68,41 +74,54 @@ const QString kFileTypeOggOpus("opus");
 const QString kFileTypeWAV("wav");
 const QString kFileTypeWavPack("wv");
 
+// Preferred picture types for cover art sorted by priority
+const std::array<TagLib::ID3v2::AttachedPictureFrame::Type, 4> kCoverArtID3v2PictureTypes = {
```

Comment:
kCoverArtID3v2PictureTypesByPriority will be more precise, but a bit long. 
How about "kPreferredID3PictureTypes" and "kPreferredVobisPictureTypes"?